pyproject.toml

```diff
@@ -1,10 +1,13 @@
 [tool.poetry]
 name = "finch-tensor"
-version = "0.2.6"
+version = "0.2.7"
 description = ""
 authors = ["Willow Ahrens <willow.marie.ahrens@gmail.com>"]
 readme = "README.md"
-packages = [{include = "finch", from = "src"}]
+packages = [
+    { include = "finch", from = "src" },
+    { include = "juliapkg.json" },
```

Comment:
IIUC, `packages` are pure-Python packages. I think what you want is [`tool.poetry.include`](https://python-poetry.org/docs/pyproject#exclude-and-include)

Reply:
(You also probably additionally need `format = ["sdist", "wheel"]`).

Reply:
Done! I included it as a `finch_julia_deps/juliapkg.json` because according to the docs you linked:
> When a wheel is installed, its includes are unpacked straight into the site-packages directory. Pay attention to include top level files and directories with common names like CHANGELOG.md, LICENSE, tests or docs only in sdists and not in wheels.